Sentinel/Classes/CrashDetection/Internal/Handlers/Preperable.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Preperable.swift
+//  MasterConnect
+//
+//  Created by Zvonimir Medak on 18.04.2024..
+//  Copyright © 2024 Signify. All rights reserved.
+//
+
+// Inspiration link: https://github.com/DebugSwift/DebugSwift
+
+import Foundation
+
+protocol Preperable {
```

Comment:
Typo; `Prepareable`, even though, I would make the name less generic (in the scope of this tool only), e..g. `CrashExceptionPrepareable`.